package.json

```diff
@@ -12,9 +12,9 @@
     "create-local": "lerna run create-local --",
     "remove-local": "lerna run remove-local --",
     "deploy-local": "lerna run deploy-local -- --version-label 0.0.1",
-    "create-remote": "lerna run create-remote -- --node http://ipfs1.dev.infra.soramitsu.co.jp:8020",
-    "remove-remote": "lerna run remove-remote -- --node http://ipfs1.dev.infra.soramitsu.co.jp:8020",
-    "deploy-remote": "lerna run deploy-remote -- --node http://ipfs1.dev.infra.soramitsu.co.jp:8020 --version-label 0.0.1"
+    "create-remote": "lerna run create-remote -- --node $DEX_GRAPHNODE_URL",
+    "remove-remote": "lerna run remove-remote -- --node $DEX_GRAPHNODE_URL",
+    "deploy-remote": "lerna run deploy-remote -- --node $DEX_GRAPHNODE_URL --version-label 0.0.1"
```

Comment:
I have no idea about `version-label` here. @toniya-klaytn Can you explain what it is?

Reply:
This is versioning the remote deployment of graph.

Reply:
This value looks be fixed. Is it okay?